File: C/1049 - Animal.c
/* 
   ==UserScript==
 @name         Animal
 @namespace    https://github.com/Ddavidi/uri-online-judge-beecrowd
 @description  URI 1049 - Animal
 @copyright    @ddavidi_
 @author       @ddavidi_	
   ==/UserScript==
*/

#include <stdio.h>
 
int main() {
 
    char animal_1[12], animal_2[12], animal_3[12];
    
    scanf("%s%s%s", &animal_1, &animal_2, &animal_3);
    
    if(animal_1[0] == 'v' && animal_2[0] == 'a' && animal_3[0] == 'c'){
        printf("aguia\n");
    }
    
    if(animal_1[0] == 'v' && animal_2[0] == 'a' && animal_3[0] == 'o'){
        printf("pomba\n");
    }
    
    
    if(animal_1[0] == 'v' && animal_2[0] == 'm' && animal_3[0] == 'o'){
        printf("homem\n");
    }
    
    if(animal_1[0] == 'v' && animal_2[0] == 'm' && animal_3[0] == 'h'){
        printf("vaca\n");
    }
    
    
    
    
    if(animal_1[0] == 'i' && animal_2[0] == 'i' && animal_3[2] == 'm'){
        printf("pulga\n");
    }
    
    if(animal_1[0] == 'i' && animal_2[0] == 'i' && animal_3[2] == 'r'){
        printf("lagarta\n");
    }
    
    
    if(animal_1[0] == 'i' && animal_2[0] == 'a' && animal_3[0] == 'h'){
        printf("sanguessuga\n");
    }
    
    if(animal_1[0] == 'i' && animal_2[0] == 'a' && animal_3[0] == 'o'){
        printf("minhoca\n");
    }
 
    return 0;
}

File: Java/1049 - Animal.java
/* 
   ==UserScript==
 @name         Animal
 @namespace    https://github.com/Ddavidi/uri-online-judge-beecrowd
 @description  URI  1049 - Animal
 @copyright    @ddavidi_
 @author       @ddavidi_	
   ==/UserScript==
*/

import java.util.Scanner;
import java.io.IOException;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        // Scanner
        Scanner scan = new Scanner(System.in);
        
        String palavra1 = scan.next();
        String palavra2 = scan.next();
        String palavra3 = scan.next();
        
        if(palavra1.equalsIgnoreCase("vertebrado")){
            if(palavra2.equalsIgnoreCase("ave")){
                if(palavra3.equalsIgnoreCase("carnivoro")){
                    System.out.println("aguia");
                }
                else{
                    System.out.println("pomba");
                }
            }
            else{
                if(palavra3.equalsIgnoreCase("onivoro")){
                    System.out.println("homem");
                }
                else{
                    System.out.println("vaca");
                }
            }
        }
        
        else{
            if(palavra2.equalsIgnoreCase("inseto")){
                if(palavra3.equalsIgnoreCase("hematofago")){
                    System.out.println("pulga");
                }
                else{
                    System.out.println("lagarta");
                }
            }
            else{
                if(palavra3.equalsIgnoreCase("hematofago")){
                    System.out.println("sanguessuga");
                }
                else{
                    System.out.println("minhoca");
                }
            }
        }
    }
}
